### DISCUSSION TITLE
v1.9.36.73 broken in almost every website

### DISCUSSION THREAD

**Discussion by Barruel** [hoothin/UserScripts#459] [Q&A]
Hi.

I'm using Pagetual over latest Tampermonkey on the Vivaldi browser.

Scripts v1.9.36.73 (1st 24th October commit) and above do nothing on most sites. I've had them working here in hoothin's github, but that's it. Everywhere else they don't work. Version 1.9.36.72 works perfectly ok as usual.

I first found the issue when browsing ebook.hunter(dot)org, but then I realized it was not working in almost every other site I've found.

**Comment by hoothin** [comment]
Try 1.9.36.75

**Reply by Barruel** [reply]
1.9.36.77 ist working.

Thank you.
